h2o-dist/index.html

```diff
@@ -554,7 +554,7 @@ <h2>Get started with H<sub>2</sub>O in 3 easy steps</h2>
             <div id="quickstart-py" style="display:none">
                 <h2>Use H<sub>2</sub>O directly from Python</h2>
 
-                <p>1. Prerequisite:    Python 2.7.x, 3.5.x, or 3.6.x</p>
+                <p>1. Prerequisite:    Python 2.7.x, 3.5.x, 3.6.x, or 3.7.x</p>
```

Comment:
can we put sth like 3.5.x to 3.7.x? especially as 3.10.x is released now and we should start to run Jenkins against it as well and I don't see us listing all versions from 3.5

Reply:
Sure... Done.